catbuffer/parser/catparser/AstPostProcessor.py

```diff
@@ -24,7 +24,7 @@ def apply_attributes(self):
 					if not hasattr(field.field_type, attribute.name):
 						raise AstException(f'field {field.name} ({field.field_type}) does not have property {attribute.name}')
 
-					setattr(field.field_type, attribute.name, attribute.value)
+					setattr(field.field_type, attribute.name, attribute.values)
```

Comment:
since qualifiers are attached to alignment property, passing down all values. this can look a bit awkward in tests, `foo.sort_key = ['abc']`.